packages/web/src/scss/components/Toast/_ToastBar.scss

```diff
@@ -48,32 +48,30 @@
     align-items: start;
 }
 
-.ToastBar__content:has(> svg:first-child) {
+.ToastBar__container:has(> svg:first-child) {
     display: grid;
     grid-template-columns: auto 1fr;
     column-gap: theme.$bar-content-gap;
 }
 
-.ToastBar:is(.ToastBar--dismissible, :has([data-spirit-dismiss='toast'])) .ToastBar__content {
+.ToastBar:is(.ToastBar--dismissible, :has([data-spirit-dismiss='toast'])) .ToastBar__container {
     align-self: center; // 4.
 }
 
-.ToastBar__message {
+.ToastBar__content {
     @include typography.generate(theme.$bar-typography);
 
     display: flex;
     flex-wrap: wrap; // 5.
     gap: theme.$bar-message-gap-y theme.$bar-message-gap-x;
 }
 
-.ToastBar__message > :is(a, button):last-child {
+.ToastBar__link {
     font-weight: 400;
 }
 
 // stylelint-disable-next-line selector-max-specificity -- Specificity is needed to precisely target the action.
```

Comment:
is this still needed? You simplified the selector so we might not need it.

Reply:
I assume this is still necessary because it styles the link font-weight. Or maybe I don't understand exactly what you mean.